### ISSUE TITLE
Default build target fails

### ISSUE THREAD

**Issue by antoniovazquezblanco** [wireghoul/graudit#9]
Graudit 1.9 source downloaded from http://www.justanotherhacker.com/projects/graudit/download.html fails to compile when make is run.

```
$ make
rm -f graudit-*.tar.gz graudit-*.zip
rm -f t/test-results/*
rm -f graudit.1 graudit.7
cat signatures/dotnet/*.db > signatures/dotnet.db
cat signatures/php/*.db > signatures/php.db
cat signatures/perl/*.db > signatures/perl.db
cat signatures/python/*.db > signatures/python.db
cat signatures/asp.db > signatures/all.db
cat signatures/c.db >> signatures/all.db
cat signatures/dotnet.db >> signatures/all.db
cat signatures/default.db >> signatures/all.db
cat signatures/jsp.db >> signatures/all.db
cat signatures/other.db >> signatures/all.db
cat signatures/perl.db >> signatures/all.db
cat signatures/php.db >> signatures/all.db
cat signatures/python.db >> signatures/all.db
cat signatures/*/xss.db > signatures/xss.db
cat signatures/*/sql.db > signatures/sql.db
cat signatures/*/exec.db > signatures/exec.db
nroff -Tascii -mandoc <graudit.in.1 >/dev/null
/bin/sh: graudit.in.1: No such file or directory
Makefile:83: recipe for target 'manpages' failed
make: *** [manpages] Error 1
```

**Comment by wireghoul** [comment]
Looks like make without an argument defaults to `make dist` which is only used to make the .tgz files from the repo. It might make more sense to have a default action, but for now you should be able to work around it using `./configure` and `make install`

**Comment by antoniovazquezblanco** [comment]
It will work for now but I'm trying to package graudit for Archlinux, this means that compilation and installation would ideally be performed in different steps.

Thanks for your time.

**Comment by wireghoul** [comment]
It's a shell script, there is no compilation. The only reason for ./configure is to allow setting the lib and bin dirs.

**Comment by antoniovazquezblanco** [comment]
You're right there's no compilation "per se" but there is file generation and other operations...
If I were you I would avoid using the configure script by making use of the "standard" DESTIR variable among others in the makefile.
https://www.gnu.org/prep/standards/html_node/Directory-Variables.html#Directory-Variables
https://www.gnu.org/prep/standards/html_node/DESTDIR.html

That should simplify the "compiation" by avoiding the use of a configuration script and by simplifying the Makefile.

**Comment by blshkv** [comment]
+1. It would help to have a clean ebuild for Pentoo Linux as well.

**Comment by wireghoul** [comment]
I'll make this change in the next update

**Comment by wireghoul** [comment]
Resolved in version 2.1, commit a9675fa3befb5cf4e78ff0f9c6f464328292aa6a

Please test and let me know if you have any issues
